.azure-pipelines/templates/run-tests.yml

```diff
@@ -22,8 +22,8 @@ steps:
 
 # Nightly base package check
 - ${{ if and(eq(parameters.test, 'true'), eq(parameters.force_base_package, 'true')) }}:
-    - script: ddev test --force-base-package --force-env-rebuild --junit ${{ parameters.check }}
-      displayName: 'Run Unit/Integration tests (no coverage, forced datadog_checks_base package)'
+    - script: ddev test --force-base-min --force-env-rebuild --junit ${{ parameters.check }}
+      displayName: 'Run Unit/Integration tests (forced install of minimum datadog_checks_base package)'
```

Comment:
Note: the test configs have no guards for which repo they get run against (e.g. `core` or `extras`), however the Azure pipeline would need to explicitly add the option for `force_base_package` to trigger it, and this defaults to `false`.